### PR TITLE
improve: Allow user to disable AWS authentication

### PR DESCRIPTION
**Pull request by grlimacan**
## Description

Optimize can be used with OpenSearch on AWS even if a AWS configuration exists in the environment.

## Related issues

closes [#14030](https://github.com/camunda/camunda-optimize/issues/14030)

### CODE REVIEW

File: optimize/util/optimize-commons/src/main/resources/service-config.yaml [hunk 1]
```diff
@@ -335,6 +335,8 @@ import:
 # a connection to it.
 opensearch:
   connection:
+    # Use AWS credentials for authentication
+    awsEnabled: ${CAMUNDA_OPTIMIZE_OPENSEARCH_AWS_ENABLED:false}
```
Comment (grlimacan) [inline optimize/util/optimize-commons/src/main/resources/service-config.yaml:339]: I am aware that the ENV var doesn't exactly match the path in the config file, but leaving the name like this to be consistent with Operate and Tasklist

Comment (RomanJRW) [reply optimize/util/optimize-commons/src/main/resources/service-config.yaml:339]: This makes sense to me 👍

File: optimize/util/optimize-commons/src/main/java/io/camunda/optimize/upgrade/os/OpenSearchClientBuilder.java [hunk 2]
```diff
@@ -111,21 +111,25 @@ private static OpenSearchTransport getAwsTransport(final ConfigurationService os
...
+      try {
+        credentialsProvider.resolveCredentials();
+        log.info("AWS Credentials can be resolved. Use AWS OpenSearch");
+        return true;
+      } catch (final Exception e) {
+        log.info("Use standard OpenSearch since AWS not configured ({}) ", e.getMessage());
+        return false;
+      }
     }
+    log.info("Not using AWS credentials for OpenSearch");
```
Comment (RomanJRW) [inline optimize/util/optimize-commons/src/main/java/io/camunda/optimize/upgrade/os/OpenSearchClientBuilder.java:126]: what about using `"AWS Credentials are disabled. Using basic auth."` to be consistent with Operate?

**Review by RomanJRW — APPROVED** [review]
LGTM, one minor suggestion

**Comment by RomanJRW** [comment]
I don't know what went wrong here, but I am guessing that this fix still needs to be merged into the release?

@grlimacan / @PHWaechtler

**Comment by grlimacan** [comment]
> I don't know what went wrong here, but I am guessing that this fix still needs to be merged into the release?
> 
> @grlimacan / @PHWaechtler

Nope, everything is ok, this is a duplicate of https://github.com/camunda/camunda/pull/21663 because it got all messed up with some branch renaming and I forgot to close this PR. Everything is merged and good, no action necessary

**Pull request closed without merging**